keywords.txt

```diff
@@ -36,7 +35,7 @@ changeI2Caddress	KEYWORD2
 setInterruptPin	KEYWORD2
 clearInterrupt	KEYWORD2
 setI2Cdelay	KEYWORD2
-getSlaveVersion	KEYWORD2
+getVersion	KEYWORD2
```

Comment:
Actually, `keywords.txt` needs to be rebuilt. Do you auto-generate it, or do it by hand? I have an auto-generator that uses ctags: I'm happy to share/contribute it.

Reply:
By hand up to now. Auto-generate sounds reasonable.

Reply:
Done. Using Makefile to generate keywords.txt